src/store/reducer.js

```diff
@@ -0,0 +1,118 @@
+import {
+  SEARCH,
+  FILTER_MOVIE,
+  VISIBLE,
+  LOAD_MOVIES,
+  FILTER_ID,
+  REMOVE_MOVIE,
+  NEW_MOVIE,
+  ADD_MOVIE,
+  RESET,
+  LOAD_MOVIES_BY_TITLE,
+  LOAD_MOVIES_BY_DATE,
+  LOAD_MOVIES_BY_RATING,
+} from "./actionTypes";
+
+const initialState = {
+  movies: [],
+  searchFilter: "",
+  filterMovie: "",
+  movieArray: [],
+  visibility: true,
+  newMovie: {
```

Comment:
you don't need this in store. All data your user types will be stored in component state and appended to the redux store in movies property on submit

Reply:
removed